File: src/resources/driver.js
db = connect("localhost:27017/app17");
coll = db.driver;
coll.insert({
    "firstName" : "Zoon",
    "middleName": "Xyhe",
    "lastName" : "Vec",
    "emailAddress": "Zoon@gmail.com",
    "password": "password001",
    "address1" : "001 Main Street",
    "address2" : null,
    "city": "Los Angeles",
    "state": "CA",
    "country" : "us",
    "postalCode" : "94035",
    "driverLicense": "F134444",
    "dlIssueState": "CA",
    "rating": Number(4.0)});
coll.insert({
      "firstName" : "Andrew",
      "middleName": "Ben",
      "lastName" : "Carlson",
      "emailAddress": "andrew@gmail.com",
      "password": "password002",
      "address1" : "002 Main Street",
      "address2" : null,
      "city": "Los Angeles",
      "state": "CA",
      "country" : "us",
      "postalCode" : "94035",
      "driverLicense": "F135555",
      "dlIssueState": "CA",
      "rating": Number(4.5)});

File: src/main/java/app17/rest/Driver.java
package app17.rest;


import javax.ws.rs.Path;

public class Driver {
    String id = null;
    String firstName;
    String middleName;
    String lastName;
    String emailAddress;
    String password;
    String address1;
    String address2;
    String city;
    String stateCode;
    String countryCode;
    String postalCode;
    String driverLicense;
    String dlIssueState;
    Number rating;
    public Driver(String firstName, String middleName, String lastName, String emailAddress, String password,
                  String address1, String address2, String city, String stateCode, String countryCode, String postalCode,
                  String driverLicense, String dlIssueState, Number rating) {
        this.firstName = firstName;
        this.middleName = middleName;
        this.lastName = lastName;
        this.emailAddress = emailAddress;
        this.password = password;
        this.address1 = address1;
        this.address2 = address2;
        this.city = city;
        this.stateCode = stateCode;
        this.countryCode = countryCode;
        this.postalCode = postalCode;
        this.driverLicense = driverLicense;
        this.dlIssueState = dlIssueState;
        this.rating = rating;
    }
    public void setId(String id) {
        this.id = id;
    }
}
